pkg/kubelet/kuberuntime/kuberuntime_sandbox_linux.go

```diff
@@ -51,7 +51,8 @@ func (m *kubeGenericRuntimeManager) calculateSandboxResources(pod *v1.Pod) *runt
 	if _, cpuRequestExists := req[v1.ResourceCPU]; cpuRequestExists {
 		cpuRequest = req.Cpu()
 	}
-	return m.calculateLinuxResources(cpuRequest, lim.Cpu(), lim.Memory())
+
+	return m.calculateLinuxResources(cpuRequest, lim.Cpu(), lim.Memory(), kubeapiqos.GetPodQOS(pod))
```

Comment:
likewise, here we can call `m.containerManager.HasExclusiveCPUs(pod)` and pass down the flag